### ISSUE TITLE
Release on PyPi

### ISSUE THREAD

**Issue by Tigge** [Tigge/antfs-cli#123]
Depends on https://github.com/Tigge/openant/issues/9

**Comment by philipwhiuk** [comment]
Is there a correct way of installing AntFS-CLI before Tigge/openant#9 is fixed - it means the current master doesn't work as it tries to download openant from PyPi.

**Comment by Tigge** [comment]
Oh, I thought I had fixed this but apparently I just fixed half of the issue. I've pushed 58951cd4dbb913ffc26f2a48c2640f87cd821a6e that should fix this.

**Comment by philipwhiuk** [comment]
Is the URL on line 47 wrong too?

**Comment by Tigge** [comment]
Yes, good catch! Fixed in afdc5ff4ef305c88c70e0fd67da52c67babd0217.

**Comment by mscansian** [comment]
Hi @Tigge, how difficult is this issue? It would be nice to install antf-cli using pip, but only if it's not too much of a hassle to you :)
